source/opcode.cpp

```diff
@@ -236,10 +236,10 @@ int32_t spvOpcodeIsPointer(const SpvOp opcode) {
   switch (opcode) {
```

Comment:
Actually, wait, I take that back.  It wouldn't be right to return true for, say, `OpTypeInt`.  Let's bake "logical" into the function name.